whosonfirst/feature.js

```diff
@@ -50,7 +50,14 @@ feature.getLocalLanguages = (feat, type = 'official') => {
 // see: https://github.com/whosonfirst-data/whosonfirst-data/issues/2154
 feature.getPlacetypeLocal = (feat) => _.flatten(
   feature.getLocalLanguages(feat, 'official')
+    // When a localized placetype isn't in a latin char set then WOF often provides
+    // a transliteration we can includesin a parenthetical, like: مقاطعة (muhafazah)
+    .map(lang => `label:${lang}_latn_x_preferred_placetype`)
+    // Otherwise prefer the local placetype name (assumes latin char set), like: provincia
     .map(lang => `label:${lang}_x_preferred_placetype`)
+    // Otherwise backfill with WOF's default English values, like: state
+    .concat(['label:eng_x_preferred_placetype'])
```

Comment:
Since this is hardcoded to English, we don't need to do a lang lookup / substitution and it can be a simple concat instead of a map.

Reply:
This new line of logic is the critical part of this PR. It allows us to achieve near 100% backfill on the new Shapefile property for coarse WOF placetypes.

Reply:
Not clear to me if we also need a map after the concat?

Reply:
The map is where the values are being pulled out of the geojson properties.

The first steps are producing a list of strings, each string represents the key of a geojson property.

The idea is that we make a list of keys, then search through the geojson to find the first matching key, the value is what we end up using for the shapefile.